src/Lexer.ts

```diff
@@ -18,7 +18,7 @@ class Lexer {
   }
 
   //  nextToken returns the next lexed token for the given input
-  public nextToken(): Token {
+  nextToken(): Token {
```

Comment:
What happened?

Reply:
its just repetitive, so i removed it. (access specifiers default to public of not mentioned)